lib/alaveteli_gettext/task_methods.rb

```diff
@@ -0,0 +1,137 @@
+# -*- encoding : utf-8 -*-
+#
+# Public: methods for the gettext rake tasks
+
+module AlaveteliGetText
+
+  module TaskMethods
+    require 'rake/file_list'
+    require 'gettext/tools/task'
+
+    def find_theme(theme)
+      unless theme
+        puts "Usage: Specify an Alaveteli-theme with THEME=[theme directory name]"
+        exit(1)
+      end
+      theme
+    end
+
+    def clean_dir(dir)
+      define_gettext_task("alaveteli",
+                          dir,
+                          Dir.glob("#{dir}/*/app.po"),
+                          "alaveteli")
+
+      Dir.glob("#{dir}/*/app.po") do |po_file|
+        GetText::Tools::MsgMerge.run("--output", po_file,
+                                     "--sort-output",
+                                     "--no-location",
+                                     "--no-wrap",
+                                     "--no-fuzzy-matching",
```

Comment:
the first 3 were, `--no-fuzzy-matching` is new (and might be incorrect) - because it's not being called as a Rake task - it's calling the msgmerge executable - it needs to be passed in directly rather than being able to rely on the task settings